src/Psr/CacheItemPool/CacheItem.php

```diff
@@ -3,24 +3,27 @@
 namespace Laminas\Cache\Psr\CacheItemPool;
 
 use DateInterval;
-use DateTimeImmutable;
 use DateTimeInterface;
-use DateTimeZone;
+use Lcobucci\Clock\Clock;
+use Lcobucci\Clock\SystemClock;
```

Comment:
As fallback solution I'd rather use a separate implementation that we ship with this package instead of a third-party as that would require that third party to actually be used.

Reply:
```suggestion
```